changelog

```diff
@@ -1,3 +1,400 @@
+      * Improved descriptions of various planets: Longjump, Buccaneer Bay, Poisonwood, Bourne, Millrace, and Solace. (@bene-dictator)
+      * Added a new category to the shipyard: Superheavy. (@Saugia)
+        * Also moved the Ikatila'ej, Korsmmanath A'awoj, Heron, and Emerald Sword to this category.
+      * Danforth's ship now also have the "timid" personality in "Wanderers: Alpha Surveillance G". (@Quantumshark)
+      * Add some diversity to the sale of Hai ships. (@MasterOfGrey)
+      * Moved the Cloaking Device is now to the "Unique" outfit category. (@czartrak)
+      * The descriptions of the Heliarch reactors now indicate that they used anti-matter catalyzed fusion technology. (@Arachi-Lover)
+      * Governments that reuse colors now refer to named "stock" colors instead of providing their own color literals. (@warp-core)
+      * Added a reference to the player's manual to the credits. (@hmglasgow)
+      * Separated planet and system definitions into their own files. (@Amazinite)
+      * Brighten some government colors to improve map readability. (@TomGoodIdea)
+      * Used custom penalties to improve Remnant and Quarg reactions to actions against some foreign governments. (@Hurleveur)
+      * Added departure distances to systems with Quarg Ringworlds. (@RisingLeaf)
+      * Void Sprites now drop Void Sprite Parts. (@Quantumshark, @Saugia)
+  * Game mechanics:
+    * New mechanics:
+      * New autoconditions:
+        * Pilot name. (@RisingLeaf)
+        * Ships, total ships, ship model, outfits, visited. (@Amazinite)
+        * Flagship planet attribute. (@rovermicrover)
+        * Several date related conditions. (@warp-core)
+        * Conditions for the number of hyperspace jumps to another planet or system. (@warp-core)
+        * Flagship attribute. (@warp-core)
+        * Flagship landed. (@Hurleveur)
+        * Entered system by and previous system. (@Hurleveur, @Amazinite)
+      * Illegal outfits and cargo:
+        * "scan brightness" attribute, increases the chance an outfit is found on a scan. (@quyykk)
+        * "scan concealment" attribute, reduces the chance an outfit is found on a scan. (@quyykk)
+        * Illegal and attrocity goods can now be defined individually for different governments. (@Hurleveur, @tibetiroka)
+      * Added the option for a hyper, scram, or jump drive to have variable fuel cost based on the mass of the ship using it. (@Ferociousfeind)
+      * Added a new "merciful" personality that causes ships to avoid attacking hostile ships that are fleeing. (@Hurleveur, @quyykk)
+      * Added "drag reduction" and "inertia reduction" attributes. (@Azure3141)
+      * Systems can now define a "departure distance", a minimum distance a ship must be from the centre to jump to another system. (@RisingLeaf, @petervdmeer, @quyykk)
+      * When taking off from a planet with an outfitter with more outfits in cargo than there is space for, store them instead of selling them. (@warp-core)
```

Comment:
```suggestion
      * When taking off from a planet with an outfitter while having more outfits in cargo than there is space for, store them instead of selling them. (@warp-core)
```